### ISSUE TITLE
[Question]How do I use a mob whitelist while allowing items to exist?

### ISSUE THREAD

**Issue by CvoidWyvern** [McJty/InControl#128]
I have this so far, it successfully stops all the mobs for any other mods from spawning, but also stops items from being dropped/dropped from blocks/thrown etc. 
```[
  {
    "mod": "pixelmon",
    "dimension": 172,
    "onjoin": true,
    "result": "allow"
  },
  {
	"dimension": 172,
    "onjoin": true,
    "result": "deny"
  },
  {
    "mod": "pixelmon",
    "dimension": 173,
    "onjoin": true,
    "result": "allow"
  },
  {
	"dimension": 173,
    "onjoin": true,
    "result": "deny"
  }
]
```
I didn't see anything in the documentation about checking if a spawn is an item or not. How would I go about changing this to allow items as well?

**Comment by CvoidWyvern** [comment]
I've managed to find a workaround in only denying entities that are hostile or passive (or from the specified mod), but I feel like this is something that either has or should have its own condition.

**Comment by McJty** [comment]
um. In Control doesn't do anything about item spawning. It is only for mob spawning

**Comment by CvoidWyvern** [comment]
Well then that's a bug, items would never spawn when using the first json file I have there, but not when only blocking hostile or passive mobs. This has happened to others playing the pack as well. The pack is at https://minecraft.curseforge.com/projects/dragonbox , 0.3.0 has the initial config that prevented item spawn, and 0.3.3 has the new config that only blocks hostile and passive. 

Boats also disappear on placing even after this, not sure if that is related.

**Comment by McJty** [comment]
Yes indeed. It's a bug and it's caused by the 'onjoin' tag. I'll fix it next release
